php-checks/src/main/java/org/sonar/php/checks/utils/TokenVisitor.java

```diff
@@ -54,9 +55,9 @@ public void visitToken(SyntaxToken token) {
   }
 
   @Nullable
-  public SyntaxToken tokenByValue(String... tokenValues) {
+  public SyntaxToken firstKeyword(Set<String> phpKeywordValues) {
     for (SyntaxToken token : tokens) {
```

Comment:
Could be nicely written in stream : 

```
tokens.stream().map(t -> token.text().toLowerCase(Locale.ROOT)).filter(phpKeywordsValues::contains).findFirst().orElse(null);
```

Reply:
@benzonico Changed with small modifications in ba6a249.